v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/database/Neo4jConnection.java

```diff
@@ -88,7 +88,7 @@ public void resetDatabase() {
     LOG.info("Resetting database");
     try {
       String database = !StringUtils.isEmpty(this.database) ? this.database : "neo4j";
-      String cypher = "CREATE OR REPLACE DATABASE $db";
+      String cypher = "CREATE OR REPLACE DATABASE $db WAIT 60 SECONDS";
```

Comment:
note: that's not available in 4.4 so we'll have to change the query in that case (in the PR that actually implement proper Neo4j 4.4 EE)

Reply:
The reason I added this is because some tests were failing because of the database not being ready in time. Let's revisit this while doing the version specific query generation work.

Reply:
Turns out, `WAIT` is supported in Neo4j 4.4 and I missed it.